format/spec.md

```diff
@@ -1861,6 +1861,10 @@ Java writes `-1` for "no current snapshot" with V1 and V2 tables and considers t
 
 Some implementations require that GZIP compressed files have the suffix `.gz.metadata.json` to be read correctly. The Java reference implementation can additionally read GZIP compressed files with the suffix `metadata.json.gz`.  
 
+### Position Delete Files with Row Data
+
+The feature allowing position delete files to include row data, which was introduced in the V2 Iceberg specification, is now deprecated in V3. The Java reference implementation will deprecate this feature in version 1.11.0 and remove it in version 1.12.0, including for V2 tables. Other implementations are encouraged to follow the same deprecation and removal schedule.
```

Comment:
A bit weird since we never introduce this feature anywhere. Also, the tense used in these sentences, `is now deprecated`, `will deprecate this feature` will become invalid over time.

Reply:
The feature is there in 1.10.0. Technically available (while it is not used anywhere as far as we know).
1.11.0 is not released yet, so it is not deprecated yet. That is why I have used future tense here.

But I'm open to suggestions, as English is not my mother language.

Reply:
I'm not sure why we would mention this here at all

Reply:
I'm OK either way but I lean towards mentioning it in the implementation notes (as done here) just because it was technically a capability enabled in the spec and even though we do not know of any writers actually producing the row data, on the offhand that some unknown implementation did produce it and wants to later use the Java library with the expectation to consume that, they can reference this historical context. 

In terms of spec wording, I think I'd remove
"Other implementations are encouraged to adopt the same deprecation and removal timeline" because I'm not sure it changes much. If in the slim chance an unknown implementation cares about producing position deletes with row data for whatever reason then they probably also want to consume it (for whatever reason)?